snap/snapcraft.yaml

```diff
@@ -95,7 +95,7 @@ apps:
   restart:
     command: bin/restart
 
-layout:
+layout: # FIXME: npm cache is on `~/.npm`
```

Comment:
Means it's a subdirectory of the program files? I might have to do some tricks here, as that might be a Read Only partition.

Do you know if it's possible to specify the cache dir?